client/src/containers/NotRegisteredPage.tsx

```diff
@@ -21,7 +36,7 @@ export default class NotRegisteredPage extends Component {
   }
 
   componentDidMount() {
-    if (this.props.geosearch && this.props.geosearch.bbl && !this.state.buildingInfo) {
+    if (this.props.geosearch && !this.state.buildingInfo) {
```

Comment:
Huh the removal of the `this.props.geosearch.bbl` in this conditional is interesting... I wonder if it might affect #340?  Not sure but the error in that issue is coming from very near this line of code.

Reply:
Actually I wonder if it makes sense to, inside this conditional, do a `if (!this.props.geosearch.bbl) rollbar.error("Geosearch result has no BBL!");` just to make sure that we're notified if our assumptions about our data types are incorrect?  I guess we could also move this to the `ApiClient.ts` file or whevever we're originally getting that `geosearch` object too, though.

Reply:
Yes, good call!